doc/man3/BIO_f_ssl.pod

```diff
@@ -241,6 +241,15 @@ a client and also echoes the request to standard output.
      exit(1);
  }
 
+ /* Added BIO_do_accept call: the first BIO_do_accept made the socket bind 
+  *  and listen and in the second BIO_do_accept will do an accept           
```

Comment:
and please remove the white space between * and text, and at the end of the line